src/vs/editor/contrib/goToDeclaration/browser/goToDeclaration.ts

```diff
@@ -217,6 +222,35 @@ export class PeekDefinitionAction extends DefinitionAction {
 	}
 }
 
+
+@editorAction
+export class GoToImplementationAction extends DefinitionAction {
+
+	public static ID = 'editor.action.goToImplementation';
+
+	constructor() {
+		super(new DefinitionActionConfig(), {
+			id: GoToImplementationAction.ID,
+			label: nls.localize('actions.goToImplementation.label', "Go to Implementation"),
+			alias: 'Go to Implementation',
+			precondition: ModeContextKeys.hasImplementationProvider,
+			kbOpts: {
+				kbExpr: EditorContextKeys.TextFocus,
+				primary: KeyMod.CtrlCmd | KeyCode.F12
```

Comment:
Ok. Visual Studio uses `alt+f12` for "go to implementation" but we already have a binding for that to `editor.action.previewDeclaration`. I don't see any default binding for cmd+f12 though, but am not sure how this would on other platforms. Any suggestions for an unused keybinding that would work on all platforms well?